packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -723,27 +723,28 @@ export class ProjectService extends BaseService {
             const projectUuid = await this.jobModel.tryJobStep(
                 jobUuid,
                 JobStepType.CREATING_PROJECT,
-                async () =>
-                    this.projectModel.create(
+                async () => {
```

Comment:
Moving logic inside `tryJobStep` to catch errors during project creation/update.